lms/static/js/commerce/views/receipt_view.js

```diff
@@ -91,15 +92,43 @@ var edx = edx || {};
                 dataType: 'json'
             }).retry({times: 5, timeout: 2000, statusCodes: [404]});
         },
+        /**
+         * Retrieve credit provider data from LMS.
+         * @param  {object} order The basket that was purchased.
+         * @return {object}                 JQuery Promise.
+         */
+        getProviderData: function (order) {
+            var provider_url = '/api/credit/v1/provider/%s/',
+                attribute_values,
+                line = order.lines[0],
+                provider_id;
+
+            this.receipt_data = order;
+
+            provider_id = this.getCreditProviderId(order);
+
+            if (provider_id) {
+                $.ajax({
+                    url: _.sprintf(provider_url, 'ASU'),
+                    type: 'GET',
+                    dataType: 'json',
+                    success: this.renderReceipt,
+                    error: this.renderError,
+                }).retry({times: 5, timeout: 2000, statusCodes: [404]});
+            } else {
+                this.renderReceipt(provider_id);
```

Comment:
if `provider_id` in `null` then it will just render the receipt page with order data without provider info.